src/vs/workbench/browser/parts/views/viewPaneContainer.ts

```diff
@@ -852,17 +852,18 @@ export class ViewPaneContainer extends Component implements IViewPaneContainer {
 			leftBorder: isPanel ? asCssVariable(PANEL_SECTION_BORDER) : undefined
 		});
 
-		const disposable = combinedDisposable(pane, onDidFocus, onDidBlur, onDidChangeTitleArea, onDidChange, onDidChangeVisibility);
-		const paneItem: IViewPaneItem = { pane, disposable };
+		const store = new DisposableStore();
+		store.add(combinedDisposable(pane, onDidFocus, onDidBlur, onDidChangeTitleArea, onDidChange, onDidChangeVisibility));
+		const paneItem: IViewPaneItem = { pane, disposable: store };
 
 		this.paneItems.splice(index, 0, paneItem);
 		assertIsDefined(this.paneview).addPane(pane, size, index);
 
 		let overlay: ViewPaneDropOverlay | undefined;
 
-		this._register(CompositeDragAndDropObserver.INSTANCE.registerDraggable(pane.draggableElement, () => { return { type: 'view', id: pane.id }; }, {}));
+		store.add(CompositeDragAndDropObserver.INSTANCE.registerDraggable(pane.draggableElement, () => { return { type: 'view', id: pane.id }; }, {}));
 
-		this._register(CompositeDragAndDropObserver.INSTANCE.registerTarget(pane.dropTargetElement, {
+		store.add(CompositeDragAndDropObserver.INSTANCE.registerTarget(pane.dropTargetElement, {
```

Comment:
@sbatten Can you please review this part? It should help with https://github.com/microsoft/vscode/issues/191899

Reply:
Am I missing where the store is disposed?

Reply:
The store is part of the `IViewPaneItem` (line 857) which is disposed when the pane is removed